admin-dev/themes/new-theme/scss/components/_form.scss

```diff
@@ -248,3 +248,7 @@
     padding-top: calc(0.375rem + 1px);
   }
 }
+
+input[type="color"].custom-color {
```

Comment:
I'm not sure this fix should be bound to a special class, if it fails on attribute page there are chances it fails on all others Can you check the other usage in the BO of this component and if they are also not integrated correctly then the fix should be generalized
You can use the `.color-widget` as the class the form group to target the correct element

Also you only changed the height of the input but for the color to take the whole height I think you also need to adapt the padding when I tested your change there was a lot of white space around the color input

Reply:
Yep, fixed height is generally a bad idea

Reply:
Not necessarily, because in this case the input field does contain the proper content it's just shrinked to a small line, probably because of the padding So at least padding needs to be fixed, and after that see if the height also needs an update

By the way I just discovered this issue https://github.com/PrestaShop/PrestaShop/issues/35181 in order return status there seems to be an identified bug on a similar input type So it's a good alternative use case to compare and check how the input can be fixed in the whole BO at once

Reply:
It's not about height though. Color picker is there and it works but just stopped looking like a color picker in develop branch. Now it's just a borderless lump of color invisible when white.

![obraz](https://github.com/PrestaShop/PrestaShop/assets/5007456/2498c51d-24b1-49e3-bc0b-3ba99245e248)